builtin/game/falling.lua

```diff
@@ -150,7 +150,12 @@ core.register_entity(":__builtin:falling_node", {
 
 		-- Rotate entity
 		if def.drawtype == "torchlike" then
-			self.object:set_yaw(math.pi*0.25)
+			if (def.paramtype2 == "wallmounted" or def.paramtype2 == "colorwallmounted")
+					and (node.param2 % 8) ~= 7 then
+				self.object:set_yaw(math.pi*0.25)
+			else
+				self.object:set_yaw(-math.pi*0.25)
+			end
```

Comment:
Condition is wrong. Try falling a `testnodes:torchlike` (the non-wallmounted one).

Should probably be:
```suggestion
			if (def.paramtype2 == "wallmounted" or def.paramtype2 == "colorwallmounted")
					and node.param2 % 8 == 7 then
				self.object:set_yaw(-math.pi*0.25)
			else
				self.object:set_yaw(math.pi*0.25)
			end
```